prow/plugins/verify-owners/verify-owners.go

```diff
@@ -41,14 +41,19 @@ import (
 
 const (
 	// PluginName defines this plugin's registered name.
-	PluginName                    = "verify-owners"
-	ownersFileName                = "OWNERS"
-	ownersAliasesFileName         = "OWNERS_ALIASES"
-	nonCollaboratorResponseFormat = `The following users are mentioned in %s file(s) but are not members of the %s org.
-
-Once all users have been added as [members](%s) of the org, you can trigger verification by writing ` + "`/verify-owners`" + ` in a comment.`
+	PluginName              = "verify-owners"
+	ownersFileName          = "OWNERS"
+	ownersAliasesFileName   = "OWNERS_ALIASES"
+	untrustedResponseFormat = `The following users are mentioned in %s file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members](%s) of the %s org. You can then trigger verification by writing ` + "`/verify-owners`" + ` in a comment.`
 )
 
+type nonTrustedReasons struct {
+	// files is a list of files they are being added in
+	files []string
+	// trigger is the reason that trigger's TrustedUser responds with for a failed trust check
+	trigger string
```

Comment:
trigger can be confusing here, maybe rename it to `reason` ?

Reply:
Called it `triggerReason` to minimize ambiguity.